Makefile

```diff
@@ -13,10 +13,13 @@ install-serverless-custom-operator: ## Install serverless with operator from IMG
 	$(call check-var,IMG)
 	make -C ${OPERATOR_ROOT} deploy apply-default-serverless-cr check-serverless-installation
 
-.PHONY: install-serverless-latest-release
-install-serverless-latest-release: ## Install serverless from latest release
+.PHONY: install-operator-latest-release
```

Comment:
It's not a good place to put operator specific targets. Put it in makefile located in `componentns/operator`.